maltd-frontend/README.md

```diff
@@ -0,0 +1,33 @@
+## MALTD Frontend
+
+In the project directory, you can run:
+
+### `npm start`
```

Comment:
should you use yarn?

Reply:
yeah we can use `yarn` instead - by running the command `yarn run start` 
yarn is an advantage for installing packages mainly over npm as it caches all installed packages making it more efficient. As for just running the application locally, it shouldn't really matter 😄 although no harm in updating for consistency